src/components/CreateMemorandum/UploadMemorandumSummary.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-card id="upload-memorandum-summary-card" flat class="rounded-0">
+    <div class="upload-memorandum-summary-header review-header">
+      <v-icon color="appDkBlue">mdi-format-list-text</v-icon>
+      <label class="upload-memorandum-title pl-2"><strong>Memorandum</strong></label>
+    </div>
+
+    <div :class="{ 'invalid-section': !getCreateMemorandumStep.valid }">
+      <div v-if="!getCreateMemorandumStep.valid" class="upload-memorandum-error-message">
```

Comment:
If this section is only displayed when `getCreateMemorandumStep.valid` is false, you can just assign the `v-if` condition and class `invalid-section` on the parent div. I don't believe there is a need for class binding.

Reply:
simplified for rules and memorandum summary components as per your suggestion

Reply:
Cool!